packages/cli/test/commands/new/entity.test.ts

```diff
@@ -0,0 +1,270 @@
+        await new Entity([entityName, '--fields', 'identifier:UUID', '--reduces', 'PostCreated'], {} as IConfig).run()
+        const renderedEntity = renderEntity(reducingEntityImports,entityName,[{ name: 'identifier', type: 'UUID' }],[{ eventName: 'PostCreated' }])
+        expect(fs.outputFile).to.have.been.calledWithMatch(entityPath, renderedEntity)
+      })
+
+      it('creates Entity with multiple fields', async () => {
+        await new Entity(
+          [entityName, '--fields', 'title:string', 'quantity:number', 'identifier:UUID'],
+          {} as IConfig
+        ).run()
+        const fields = [
+          { name: 'title', type: 'string' },
+          { name: 'quantity', type: 'number' },
+          { name: 'identifier', type: 'UUID' },
+        ]
+        const renderedEntity = renderEntity(defaultEntityImports,entityName,fields,[])
+        expect(fs.outputFile).to.have.been.calledWithMatch(entityPath, renderedEntity)
+      })
+
+      it('creates Entity with multiple fields reducing PostCreated', async () => {
+        await new Entity(
+          [entityName, '--fields', 'title:string', 'quantity:number', 'identifier:UUID', '--reduces', 'PostCreated'],
+          {} as IConfig
+        ).run()
+        const fields = [
+          { name: 'title', type: 'string' },
+          { name: 'quantity', type: 'number' },
+          { name: 'identifier', type: 'UUID' },
+        ]
+        const renderedEntity = renderEntity(reducingEntityImports,entityName,fields,[{ eventName: 'PostCreated' }])
+        expect(fs.outputFile).to.have.been.calledWithMatch(entityPath, renderedEntity)
+      })
+
+      it('creates Entity with multiple fields reducing PostCreated and CommentCreated', async () => {
+        await new Entity(
+          [
+            entityName,
+            '--fields',
+            'title:string',
+            'quantity:number',
+            'identifier:UUID',
+            '--reduces',
+            'PostCreated',
+            'CommentCreated',
+          ],
+          {} as IConfig
+        ).run()
+        const fields = [
+          { name: 'title', type: 'string' },
+          { name: 'quantity', type: 'number' },
+          { name: 'identifier', type: 'UUID' },
+        ]
+        const renderedEntity = renderEntity(reducingTwoEntityImports,entityName,fields,[{ eventName: 'PostCreated' }, { eventName: 'CommentCreated' }])
+        expect(fs.outputFile).to.have.been.calledWithMatch(entityPath, renderedEntity)
+      })
+    })
+
+    describe('displays an error', () => {
+      it('with empty Entity name', async () => {
+        replace(console, 'error', fake.resolves({}))
+        await new Entity([], {} as IConfig).run()
+        expect(fs.outputFile).to.have.not.been.calledWithMatch(entitysRoot)
+        expect(console.error).to.have.been.calledWith(
+          "You haven't provided an entity name, but it is required, run with --help for usage"
```

Comment:
As error messages and logs could easily change without affecting the correctness of the application, I would suggest writing expectations on those messages using regular expressions that checks only the main part of the message, for example in this case it would be:
```suggestion
          /You haven't provided an entity name/
```
This way, it is less cumbersome to do code changes and refactors that only affect these messages.

What do you think? 

By the way, this message also applies to other similar expectations in this PR

Reply:
Suggested changes applied! You are right!